model/__init__.py

```diff
@@ -298,6 +298,14 @@ def secrets_server(self, cfg_name=None):
     def protecode(self, cfg_name=None):
         return self.cfg_factory.protecode(self._default_name('protecode', cfg_name))
 
+    def iter_configs(self, cfg_type_name):
+        if not cfg_type_name in self.raw:
+            raise ValueError("No config of type '{c}' in ConfigSet.".format(c=cfg_type_name))
+        if 'config_names' in self.raw[cfg_type_name]:
+            yield from self.raw[cfg_type_name]['config_names']
```

Comment:
return the actual cfg_elements rather than just their names (this is what the method name suggests)